source/ui/BioXAS/BioXASMainXASScanConfigurationView.cpp

```diff
@@ -36,6 +37,8 @@ BioXASMainXASScanConfigurationView::BioXASMainXASScanConfigurationView(BioXASMai
 
 	connect(scanName_, SIGNAL(editingFinished()), this, SLOT(onScanNameEdited()));
 	connect(configuration_, SIGNAL(nameChanged(QString)), scanName_, SLOT(setText(QString)));
+    connect(configuration_, SIGNAL(totalTimeChanged(double)), this, SLOT(onEstimatedTimeChanged()));
```

Comment:
Technically, since the time is being passed as the `double`, you could make the `onEstimatedTimeChanged()` method take a double as well.  That way, you don't need to ask the configuration for the time again.